frontend/src/features/settings/userSettingsSlice.ts

```diff
@@ -15,15 +16,22 @@ export const fetchUserSettings = createAsyncThunk(
 export const userSettingsSlice = createSlice({
   name: 'settings',
   initialState: userSettingsInitialState,
-  reducers: {},
+  reducers: {
+    // Replace all user's settings of all polls by new ones.
+    replaceSettings: (state, action) => {
+      state.settings = action.payload;
+    },
+  },
   extraReducers: (builder) => {
-    /**
-     * Replace all user's settings by new ones.
-     */
+    // Replace all user's settings of all polls by new ones.
```

Comment:
Side question, not directly related to this PR:
The settings are currently persisted when the user logs out. Maybe they would be expected to be removed from local storage on logout?

Reply:
Good idea, let's do it.